_posts/en/newsletters/2024-05-31-newsletter.md

```diff
@@ -0,0 +1,237 @@
+
+- [Does a P2PKH transaction send to the hash of the uncompressed key or the compressed key?]({{bse}}122875)
+  Pieter Wuille notes that both compressed and uncompressed public keys can be
+  used resulting in distinct P2PKH addresses, adding that P2PKH only supports
+  compressed public keys and P2TR uses [x-only public keys][topic X-only public keys].
+
+- [What are different ways to broadcast a block to the Bitcoin network?]({{bse}}122953)
+  Pieter Wuille outlines 4 ways to announce blocks on the P2P network: using
+  [BIP130][], using [BIP152][], sending [unsolicited `block` messages][], and the older
+  `inv` / `getdata` / `block` message flow.
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [LND v0.18.0-beta][] is the latest major release of this popular LN
+  node implementation.  According to its [release notes][lnd rn],
+  experimental support is added for _inbound routing fees_ (see
+  [Newsletter #297][news297 inbound]), pathfinding for [blinded
+  paths][topic rv routing] is now available, [watchtowers][topic
+  watchtowers] now support [simple taproot channels][topic simple
+  taproot channels], and sending encrypted debugging information is now
+  streamlined (see [Newsletter #285][news285 encdebug]), with many other features also
+  added and many bugs fixed.
+
+- [Core Lightning 24.05rc2][] is a release candidate for the next major
+  version of this popular LN node implementation.
+
+## Notable code and documentation changes
+
+_Notable recent changes in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo],
+[Bitcoin Inquisition][bitcoin inquisition repo], and [BINANAs][binana
+repo]._
+
+- [Bitcoin Core #29612][] updates the serialization format of the UTXO set dump
+  output through the `dumptxoutset` RPC. This results in a 17.4% space
+  optimization. The `loadtxoutset` RPC now expects the new format when loading
+  the UTXO set dump file; the old format is no longer supported. See Newsletters
+  [#178][news178 txoutset] and [#72][news72 txoutset] for previous references to
+  `dumptxoutset`.
+
+- [Bitcoin Core #27064][] changes the default data directory on Windows from
+  `C:\Users\Username\AppData\Roaming\Bitcoin` to
+  `C:\Users\Username\AppData\Local\Bitcoin` on fresh installs only.
+
+- [Bitcoin Core #29873][] introduces a 10 kvB data weight limit for
+  [Topologically Restricted Until Confirmation (TRUC)][topic v3 transaction
+  relay] transactions (v3 transactions) to reduce the potential cost of
```

Comment:
Sorry for late review but I think this is quite a significant omission.

The PR makes it cheaper to defend against pinning attacks, however the original text sounds like it makes attacking cheaper.

@glozow says:

> It lowers potential replacement cost (i.e. harder to do Rule 3 pinning via gigantic transaction)

```suggestion
  relay] transactions (v3 transactions) to reduce the potential cost of mitigation against
```

Reply:
Taking this suggestion